packages/lesswrong/server/search/elastic/ElasticExporter.ts

```diff
@@ -102,8 +145,9 @@ class ElasticExporter {
         index: newIndexName,
         name: aliasName,
       });
-      await client.indices.delete({
+      await client.indices.deleteAlias({
         index: oldIndexName,
+        name: aliasName,
```

Comment:
This is currently the case - there's about 45 seconds when configuring an index where it still works, but there's less documents available whilst they're copied over.

I think changing this would be out of scope for this PR.

Reply:
Can you document this? It seems important for future developers to understand.